.github/workflows/db2_online.yml

```diff
@@ -1,6 +1,13 @@
 name: Db2 online tests
 
 on: [pull_request]
+  # trigger on PR, but only on main (dev), stable, and maintenance branches 
```

Comment:
We'll want to back out these changes before merging (either in remove-opengis, or when that is merged into main). We don't want to have one more bit to maintain at every major release.

Reply:
Yes of course; I just needed faster feedback and dislike wasting resources